common/src/Model/GroupNode.cpp

```diff
@@ -72,53 +73,52 @@ namespace TrenchBroom {
                 }
             }
 
-            // In parallel, copy the contents of nodesToClone, transformed, into origNodeAndTransformedContents
-            using NodeContentType = std::variant<std::monostate, Group, Entity, Brush, BezierPatch>;
-            using NodeAndContent = std::pair<const Node*, NodeContentType>;
+            using TransformResult = kdl::result<std::pair<const Node*, NodeContents>, BrushError>;
 
-            auto origNodeAndTransformedContents = std::vector<NodeAndContent>{nodesToClone.size(), std::make_pair(nullptr, std::monostate{})};
```

Comment:
You lost my self documenting variable name here.. it's important that it's the original node, and the transformed contents.

Reply:
Oh ok, it appears below.